docs/pages/tutorial/create-a-modal.mdx

```diff
@@ -334,7 +334,7 @@ import MaterialIcons from '@expo/vector-icons/MaterialIcons';
 
 type Props = {
   isVisible: boolean;
-  children: React.ReactNode;
+  children?: React.ReactNode;
   onClose: () => void;
 };
```

Comment:
Hey @iBiziCode, thanks for catching this problem! 👍 

Nothing is wrong with your fix, but let's use `PropsWithChildren` helper type to spread a pattern:

```rs
type Props = PropsWithChildren<{
  isVisible: boolean;
  onClose: () => void;
}>;
```

Reply:
Hey @Simek, Thank you for the new insight. We can do that.